cartography/intel/aws/__init__.py

```diff
@@ -207,6 +207,12 @@ def start_aws_ingestion(neo4j_session: neo4j.Session, config: Config) -> None:
         common_job_parameters,
     )
 
+    run_analysis_job(
```

Comment:
Because the analysis job itself relies on `AWS_ID` for the current AWS account ID, you will want to put this in `_sync_one_account` and not in `start_aws_ingestion`.